package.json

```diff
@@ -22,7 +22,7 @@
     "build:dist": "node bin/check-nvmrc.js && gulp build:dist --destination 'dist' && npm run test:build:dist",
     "test": "standard && gulp test && npm run test:app && npm run test:components && npm run test:generate:readme",
     "test:app": "jest app/__tests__/app.test.js --forceExit # express server fails to end process",
-    "test:components": "jest src/",
+    "test:components": "jest src/ && jest tasks/gulp/__tests__/check-individual-components-compile.test.js",
```

Comment:
Should this run all tests under `tasks/gulp/__tests__` rather than a specific case?

Reply:
Some of the task tests run 'after' tasks.

Reply:
👍